Doc/library/mmap.rst

```diff
@@ -370,6 +370,8 @@ MAP_* Constants
           MAP_ANONYMOUS
           MAP_POPULATE
           MAP_STACK
+          MAP_ALIGNED_SUPER
+          MAP_CONCEAL
 
     These are the various flags that can be passed to :meth:`mmap.mmap`. Note that some options might not be present on some systems.
```

Comment:
Please describe that MAP_ALIGNED_SUPER and MAP_CONCEAL are only available at OpenBSD.